src/features/user/ManagePayouts/index.tsx

```diff
@@ -53,14 +53,11 @@ export default function ManagePayouts({
     if (!payoutMinAmounts) {
       try {
         const res = await getRequest<Payouts.PayoutMinAmounts>(
-          '/app/payoutMinAmounts',
-          handleError
+          '/app/payoutMinAmounts'
         );
-        if (res && !res['error_code']) {
-          setPayoutMinAmounts(res);
-        }
+        setPayoutMinAmounts(res);
```

Comment:
We should not check the properties in the response because according to standard practices it is considered that the data structure we get is correct, this should be done while structuring APIs itself.

In earlier code we were checking for success response and error response, the API handler function earlier was not handling both of them properly since it was sending both success response and error response.

The current implementation handles it gracefully.